solidity/contracts/libraries/operator/Groups.sol

```diff
@@ -19,6 +21,10 @@ library Groups {
     // before using the value.
     uint256 constant GROUP_INDEX_FLAG = 1 << 255;
 
+    uint256 constant ONE_MONTH = 86400 * 30;
```

Comment:
I think we can do `= 30 days` instead `= 86400 * 30`
These are available time units: seconds, minutes, hours, days, weeks and years